node/resolvers/search/facets.ts

```diff
@@ -153,6 +146,18 @@ export const resolvers = {
 
     name: formatTranslatableProp<SearchFacetCategory, 'Name', 'Id'>('Name', 'Id')
   },
+  Facet: {
+    values: (facet: Facet, args: FacetValuesArgs) => {
+      const { values } = facet
+      if (args.from === undefined) {
+        return values
+      }
+      return values.slice(args.from, args.to)
```

Comment:
se a pessoa só passar `to`, nao da problema talvez? se der, talvez isso deve resolver

```suggestion
      return values.slice(args.from ?? 0, args.to)
```